sourcegraph/sourcegraph
# feat(licensing) add link to license key for anomaly check slack message

Part of https://github.com/sourcegraph/sourcegraph/issues/52507.

This PR:
- feat(licensing): add link to license key for anomaly check slack message
- style(SiteAdminProductLicenseNode): update UI to expand/collapse additional info

## Test plan
- Check the diff
- `sg start dotcom`
- Go to any product subscription page

## Demo

https://github.com/sourcegraph/sourcegraph/assets/6717049/447642ae-a55c-42db-8fb1-54fd5a38dfb5
